scripts/supercloud/run_predicators_sidelining_experiments.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+START_SEED=456
+NUM_SEEDS=10
+FILE="scripts/supercloud/submit_supercloud_job.py"
+# Note: this script is meant to be run first, to completion, with
+# RUN_LOAD_EXPERIMENTS=false, then rerun with RUN_LOAD_EXPERIMENTS=true.
+RUN_LOAD_EXPERIMENTS=false
+NUM_TRAIN_TASKS="50"
+ALL_ENVS=(
+    "repeated_nextto_painting"
+    "screws"
+    "repeated_nextto_single_option"
+    "painting"
```

Comment:
probs worth running this one locally on one seed to see what happens, if you haven't already

Reply:
ran locally and it solved 50/50, BUT it ended up learning sidelined operators for placing (which is definitely unexpected). Probably something we should look into!

Reply:
can you paste the operators?

Reply:
(they're on slack)